src/content/docs/ru/tutorial/2-pages/2.mdx

```diff
@@ -0,0 +1,197 @@
+---
+type: tutorial
+title: Первый пост в блоге с использованием Markdown
+description: |-
+  Обучающий курс: Создание первого блога Astro —
+  Добавление страниц с использованием Markdown на ваш сайт
+i18nReady: true
+---
+import Badge from '~/components/Badge.astro';
+import Checklist from '~/components/Checklist.astro';
+import Blanks from '~/components/tutorial/Blanks.astro';
+import Box from '~/components/tutorial/Box.astro';
+import MultipleChoice from '~/components/tutorial/MultipleChoice.astro';
+import Option from '~/components/tutorial/Option.astro'
+import PreCheck from '~/components/tutorial/PreCheck.astro';
+
+
+
+Теперь, когда вы построили страницы, используя файлы `.astro`, давайте создадим несколько записей в блоге, используя файлы `.md`!
+
+<PreCheck>
+  - Создайть новую папку и создайть новую запись
+  - Напишите некоторое содержимое Markdown
+  - Добавить ссылки на ваши записи на страницу вашего блога
+</PreCheck>
+
+## Создайте свой первый файл `.md`
+
+1. Создайте новый каталог в `src/pages/posts/`.
+
+2. Добавьте новый (пустой) файл `post-1.md` внутри вашей новой папки `/posts/`.
+
+3. Найдите эту страницу в браузерном предпросмотре, добавив `/posts/post-1` в конец вашего существующего URL-адреса предварительного просмотра.
+
+    (например, `localhost:3000/posts/post-1`)
+
+4. Измените URL-адрес предварительного просмотра в браузере, чтобы просмотреть `/posts/post-2` вместо этого. (Это страница, которую вы еще не создали.)
+
+    Обратите внимание на различный вывод при предварительном просмотре «пустой» страницы и страницы, которой еще не существует. Это поможет вам устранить проблемы в будущем.
+
+## Напишите содержимое Markdown
+
+1. Скопируйте или введите следующий код в файл `post-1.md`
+
+    ```markdown title="src/pages/posts/post-1.md»
```

Comment:
Typo causes the title to not render.
```suggestion
    ```markdown title="src/pages/posts/post-1.md"
```